charts/posthog/templates/test-templates.yaml

```diff
@@ -6,6 +6,8 @@ data:
 {{- include "snippet.clickhouse-env" . | nindent 2 }}
 {{- else if .Values.testTemplates.postgresqlEnvTest -}}
 {{- include "snippet.postgresql-migrate-env" . | nindent 2 }}
+{{- else if .Values.testTemplates.kafkaEnvTest -}}
```

Comment:
Do we need this? Unless it’s used in process of development I’d propose removing